config/symbols.pspeu.stwrp.txt

```diff
@@ -107,3 +107,5 @@ g_eBreakableExplosionTypes = 0x0924BCC8;
 D_80180608 = 0x0924CCD0;
 D_80180530 = 0x0924CCD8;
 D_80180648 = 0x0924CCE0;
+unk15_rot = 0x80180FF8;
```

Comment:
I"m thinking these addresses are off.
```
mipsel-linux-gnu-ld: build/pspeu/src/st/wrp_psp/e_misc.c.o: in function `EntityUnkId15':
(.text+0x7c): undefined reference to `unk15_rot'
mipsel-linux-gnu-ld: (.text+0x80): undefined reference to `unk15_rot'
mipsel-linux-gnu-ld: (.text+0xac): undefined reference to `unk15_yVel'
mipsel-linux-gnu-ld: (.text+0xb0): undefined reference to `unk15_yVel'
```
You can look at the original asm here for the addresses https://decomp.me/scratch/ez7sR